packages/daimo-api/src/cron.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  amountToDollars,
+  getAccountName,
+  guessTimestampFromNum,
+} from "@daimo/common";
+import {
+  daimoChainFromId,
+  daimoPaymasterAddress,
+  entryPointABI,
+} from "@daimo/contract";
+import { CronJob } from "cron";
+import { Constants } from "userop";
+import { Hex, formatEther } from "viem";
+
+import { CoinIndexer, TransferLog } from "./contract/coinIndexer";
+import { NameRegistry } from "./contract/nameRegistry";
+import { ViemClient, chainConfig } from "./env";
+import { Telemetry } from "./telemetry";
+
+export class Crontab {
```

Comment:
i'd definitely prefer this in a separate repo

not good to overload the API server with other tasks

but OK for now